packages/cli/src/command-implementations/deploy-command.ts

```diff
@@ -51,6 +51,11 @@ export class DeployCommand extends ProjectCommand<DeployCommandDefinition> {
     const readmeFileContent = await this._readMediaFile('readme', readmeRelativeFilePath)
 
     const integration = await api.findIntegration({ type: 'name', name, version })
+    if (integration && !integration.workspaceId) {
+      throw new errors.BotpressCLIError(
+        `Integration ${integrationDef.name} v${integrationDef.version} is already deployed publicly in another workspace.`
```

Comment:
I think `Public integration` is more clear than `deployed publicly` (the notion of `deployed publicly` does not exist)

```suggestion
        `Public integration ${integrationDef.name} v${integrationDef.version} is already deployed in another workspace.`
```